atomic_reactor/inner.py

```diff
@@ -404,6 +445,110 @@ def _update(data):
         return new_data
 
 
+@dataclass
+class ImageBuildWorkflowData(ISerializer):
+    """Manage workflow data.
+
+    Workflow data is those data which are generated values by plugins through
+    the whole build workflow (pipeline) and must be shared in every build tasks.
```

Comment:
Nice! :tada: 

I love how much data you were able to remove from DockerBuildWorkflow. It almost looks like a sane class now :smile: